aggsender/block_notifier_polling_test.go

```diff
@@ -193,7 +193,7 @@ func newBlockNotifierPollingTestData(t *testing.T, config *ConfigBlockNotifierPo
 	if config == nil {
 		config = &ConfigBlockNotifierPolling{
 			BlockFinalityType:     etherman.LatestBlock,
-			CheckNewBlockInterval: time.Second,
+			CheckNewBlockInterval: 0,
 		}
 	}
 	EthClientMock := mocks.NewEthClient(t)
```

Comment:
Nitpick (rename the other places in this scope as well, I couldn't add comment):
```suggestion
	ethClientMock := mocks.NewEthClient(t)
```